scenarios/llm/run.sh

```
@@ -0,0 +1,147 @@
+    sleep 1
+  done
+
+  echo
+  echo "-- Step 2: inject the outage (scale Ollama to 0 replicas) --"
+  kubectl -n "${NAMESPACE}" scale deployment/ollama --replicas=0
+  echo "Ollama scaled to 0. Waiting 15s for it to actually go down..."
+  sleep 15
+
+  echo
+  echo "-- Step 3: send traffic during the outage --"
+  for i in 1 2 3; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)